src/components/homepage/Home.jsx

```diff
@@ -1,16 +1,21 @@
 import React from "react";
 import TrendMusics from "./TrendMusics";
+<<<<<<< HEAD
```

Comment:
ACHTUNG !
There are conflict artefacts in this file !
Nobody read this code before me -_-